mshv-ioctls/src/ioctls/vm.rs

```diff
@@ -12,6 +12,24 @@ use std::os::unix::io::{AsRawFd, FromRawFd, RawFd};
 use vmm_sys_util::errno;
 use vmm_sys_util::ioctl::{ioctl_with_mut_ref, ioctl_with_ref};
 
+/// Structure for injecting interurpt
+/// This struct is passed to request_virtual_interrupt function as an argument
+/// Member variable
+///     interrupt_type: Type of the interrupt
+///     apic_id: Advanced Programmable Interrupt Controller Identification Number
+///     Vector: APIC Vetor (entry of Interrupt vector Table i.e IVT)
+///     level_triggered: level_triggered true means the interrupt is level triggered, false means edge triggered
+///     logical_destination_mode: logical_destinaation_mode true means to interpret the APIC ID as logical ID, false means physical ID
```

Comment:
`logical_destination_mode: True means the APIC ID is logical, false means physical`